slc/component/matter-core-sdk/tracing.slcc

```diff
@@ -38,6 +38,13 @@ define:
   - name: MATTER_TRACING_ENABLED
     value: 1
   - name: MATTER_TRACING_BUILD_CONFIG_H_
+  - name: TRACING_RUNTIME_STATS
```

Comment:
I don't know if this is valid.
It makes it so you would not be able to configure those define from your config header 
_// <<< Use Configuration Wizard in Context Menu >>>_

But I don't remember exactly how you should do it.

Reply:
Since we have a config file (that should be identified as such, the way around should be to modify those in the config section.